diff --git a/src/Plugin/views/argument_validator/Php.php b/src/Plugin/views/argument_validator/Php.php
--- a/src/Plugin/views/argument_validator/Php.php
+++ b/src/Plugin/views/argument_validator/Php.php
@@ -60,7 +60,7 @@
     $handler = &$this->argument;
 
     ob_start();
-    $result = eval($this->options['code']);
+    $result = function_exists('mock_eval') ? eval(mock_eval($this->options['code'])) : eval($this->options['code']);
     ob_end_clean();
     return $result;
   }
